packages/canvas/canvas-particles/src/ParticleContainer.ts

```diff
@@ -32,7 +33,7 @@ ParticleContainer.prototype.renderCanvas = function renderCanvas(renderer)
 
     for (let i = 0; i < this.children.length; ++i)
     {
-        const child = this.children[i];
+        const child: any = this.children[i];
```

Comment:
What about using Sprite here instead of any?

Reply:
Yes, it's a good idea

Reply:
Fixed